idaes/logger.py

```diff
@@ -297,6 +298,7 @@ def run(self):
             self.log_value()
             self.stop.wait(self.sleep)
             if self.stop.isSet():
+                time.sleep(2)
```

Comment:
we can probably update 2 to self.sleep

Reply:
I tried that, but got a test failure, I assume because the output was logged in a different order.

Reply:
What test failed?

Reply:
Ah, yeah it looks like the delay for self.sleep (1 s I think) is too short still.  I'll create an issue for potentially revisiting this that explains the problem more thoroughly.  I think we should merge this now to avoid the problem where the solver fails and we don't know why due to missing output, and we can discuss if there is a better solution later.